test/linalg/graphmatrices.jl

```diff
@@ -166,36 +166,20 @@ export test_adjacency, test_laplacian, test_accessors, test_arithmetic, test_oth
         @test norm(z) ≈ 0.0 atol=1e-8
     end
 
-    begin
-        n = 10
-        mat = sparse(spones(sprand(n,n,0.3)))
-        begin
-            test_adjacency(mat)
-        end
-
-        begin
-            test_laplacian(mat)
-        end
 
-        begin
-            test_accessors(mat, n)
-        end
-    end
+    n = 10
```

Comment:
What is motivating these changes? The begin end blocks here are to isolate the scopes of the tests to make sure that state doesn't bleed from one test to the next.